apps/condo/domains/common/components/UseDeskWidget.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import getConfig from 'next/config'
+
+const UseDeskWidget: React.FC = () => {
+
+    const { publicRuntimeConfig:{ UseDeskWidgetId } } = getConfig()
+
+    return UseDeskWidgetId ?
+        <script async src={`https://lib.usedesk.ru/secure.usedesk.ru/${UseDeskWidgetId}.js`}></script> : null
```

Comment:
Maybe it's better to use the current schema?
```suggestion
        <script async src={`//lib.usedesk.ru/secure.usedesk.ru/${UseDeskWidgetId}.js`}></script> : null
```